### DISCUSSION TITLE
ADOT Helm chart for EKS on fargate

### DISCUSSION THREAD

**Discussion by pribanerjee** [aws-observability/aws-otel-helm-charts#52] [General]
Can someone please help provide where can I find the helm chart to deploy ADOT collector for EKS on fargate?

The one present here [adot-exporter-for-eks-on-ec2](https://github.com/aws-observability/aws-otel-helm-charts/tree/main/charts/adot-exporter-for-eks-on-ec2) is for EKS on EC2.

**Comment by xgt001** [comment]
I created a very barebones stripped down fork of this repo specifically for this - I needed to get a feel of containerInsights for Fargate on EKS
https://github.com/xgt001/aws-otel-helm-charts-fargate 
Hope this is a bit helpful :)
